tests/garage/tf/models/test_cnn_model.py

```diff
@@ -60,6 +61,54 @@ def test_output_value(self, filter_sizes, in_channels, out_channels,
 
         assert np.array_equal(output, expected_output)
 
+    @params(((1, ), (3, ), (32, ), (1, ), (1, 1),
```

Comment:
is this weird formatting from yapf? if so, please make use `# yapf: disable` to stop it